src/app/frontend/uicomponents/helpsection/helpsection.scss

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.kd-help-section > * {
+  width: 50% !important;
```

Comment:
Doesn't flex help here?

Make kd-help-section container flex or use template in it like:
```html
<div layout="row">
  <ng-transclude/>
</div>
```